t/22-svirt.t

```diff
@@ -1034,4 +1034,136 @@ subtest 'get_wait_still_screen_on_here_doc_input' => sub {
     is($svirt->get_wait_still_screen_on_here_doc_input({}), 0, 'wait_still_screen on here doc is not set for kvm');
 };
 
+subtest 'Routine consoles::sshVirtsh::provide_image_vmware_in_ds, shell script run' => sub {
+    set_var(VMWARE_NFS_DATASTORE_DEBUG => 1);
+
+    my $my_test_basedir = tempdir($dir . '/tb_XXXX');    # base subdir of testing, default /vmfs/volumes.
+    set_var(VIRSH_OPENQA_BASEDIR => $my_test_basedir);
+
+    # orig. like: /vmfs/volumes/openqa/hdd/img.vmdk.xz
+    my $nfs_ds = 'openqa200';
+    set_var(VMWARE_NFS_DATASTORE => $nfs_ds);
+    my $my_test_dir = path($my_test_basedir, $nfs_ds)->make_path;
+    my $my_test_dir_hdd = path($my_test_dir, 'hdd')->make_path;
+    my $my_test_dir_iso = path($my_test_dir, 'iso')->make_path;
+
+    # dest: /vmfs/volumes/Datastore1/openQA/img.vmdk.xz
+    my $ds = 'datastore200';
+    set_var(VMWARE_DATASTORE => $ds);
+    my $vmware_openqa_datastore = path($my_test_basedir, $ds, 'openQA')->make_path;
+
+    # images
+    my $input_file = path('vmware-mock200-image.vmdk');
+    my $input_file_xz = path($input_file . '.xz');
+
+    # presets:
+
+    # origin dir: create dummy file and compress .xz
+    my $file = path($my_test_dir, $input_file)->touch;
+    qx(xz -f --compress --keep $file);
```

Comment:
Maybe a good idea to remove this at the end of the test.

Reply:
The full local temporary directory is automatically deleted at end of the test and all used files will disappear.
Unless, do you meant to remove those test code lines?